eZ/Publish/API/Repository/Tests/SearchService/SearchServiceFulltextEmbedTest.php

```diff
@@ -0,0 +1,184 @@
+<?php
+
```

Comment:
The correct file header is as follows (pay attention to blank lines as well, please):
``` php
<?php

/**
 * @copyright Copyright (C) eZ Systems AS. All rights reserved.
 * @license For full copyright and license information view LICENSE file distributed with this source code.
 */
declare(strict_types=1);

namespace eZ\Publish\API\Repository\Tests\SearchService;
```

There's a pending PR enforcing this via php-cs-fixer, still work in progress though, sorry :)